frontend/src/featureFlags/WithFeatureFlags.tsx

```diff
@@ -0,0 +1,42 @@
+import { useEffect, useState } from "react";
+import { FlagsProvider } from "flagged";
+
+import FetchClient from "../app/utils/api";
+
+type WithFeatureFlagsProps = {
+  children: JSX.Element;
+};
+
+const SR_APP_FEATURES = "sr-app-features";
+const api = new FetchClient(undefined, { mode: "cors" });
```

Comment:
Our pattern elsewhere has been to specify the root path at the `FetchClient` level (see [`AccountCreationApiService`](https://github.com/CDCgov/prime-simplereport/blob/c3e6b2dc330021e425e01657eaee613c51f9e11c/frontend/src/app/accountCreation/AccountCreationApiService.ts#L5-L5), could you edit this `FetchClient` to operate the same way?

Reply:
Done!